security/keycloak/src/test/java/io/quarkus/ts/security/keycloak/DevModeKeycloakDevServiceUserExperienceIT.java

```diff
@@ -15,7 +15,7 @@
 @QuarkusScenario
 public class DevModeKeycloakDevServiceUserExperienceIT {
 
-    private static final String KEYCLOAK_VERSION = "18.0";
+    private static final String KEYCLOAK_VERSION = "19.0.1";
```

Comment:
In this scenario we are forcing a different version than the default one, so could we use for example Keycloak 19.0.0 ?

Reply:
Yes. Changed to 19.0.0